codegen/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "loopback-sdk-ios-codegen",
```

Comment:
I would personally move this file to project's root and rename the module to `loopback-sdk-ios`.